wgsl/index.bs

```diff
@@ -8608,6 +8615,30 @@ path: syntax/location_attr.syntax.bs.include
 
 </table>
 
+<pre class=include>
```

Comment:
Right now this is part of the location subsection. This should be it's own subsection (alphabetically sorted).

Reply:
It still needs to be sorted alphabetically.